apps/tup-cms/src/apps/dashboard/views.py

```diff
@@ -1,13 +1,27 @@
-from curses.ascii import HT
 from django.http import HttpResponse
 from django.conf import settings
 from django.template import loader
+from django.shortcuts import redirect
 from django.conf import settings
+from django.contrib.auth import authenticate, login
 
 
-def AddedView(request):
+def LoginView(request):
```

Comment:
Is this already covered by Django's built in LoginView?

https://docs.djangoproject.com/en/4.1/topics/auth/default/#django.contrib.auth.views.LoginView